.idea/workspace.xml

```diff
@@ -0,0 +1,149 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
We can't commit this file. I'm guessing it was added by mistake. We can change the project `.gitignore` to prevent this in the future to ignore the `.idea/` directory entirely.

Reply:
You may remove this file by running `git rm --cached -- .idea/workspace.xml`

We can prevent this from happening again by adding .idea/ to https://github.com/mongodb/mongo-python-driver/blob/master/.gitignore (you may do that in this PR as well)